apps/web/pages/booking/[uid].tsx

```diff
@@ -345,7 +345,7 @@ export default function Success(props: SuccessProps) {
           eventName={eventName}
           startTime={bookingInfo.startTime}
           endTime={bookingInfo.endTime}
-          organizer={bookingInfo.user}
+          organizer={bookingInfo.user[0]}
```

Comment:
Assume that the organizer of this booking is the first user in the user array. Although @emrysal would this tie in with your PR that added `host`?